app.js

```diff
@@ -339,13 +340,26 @@ Promise.all([compilerFinder.find(), aws.initConfig(awsProps)])
             webServer.use(express.static(defArgs.staticDir, {maxAge: staticMaxAgeSecs * 1000}));
         }
 
+        morgan.token('gdpr_ip', req => utils.anonymizeIp(req.ip));
+
+        // Based on combined format, but: GDPR compilant IP, no timestamp & no unused fields for our usecase
```

Comment:
I  linked to https://www.theregister.co.uk/2018/04/24/ietf_gdpr_compliance_advice/ in my [comment](https://github.com/mattgodbolt/compiler-explorer/issues/972#issuecomment-400559784) in #972 . That's where the suggestions for GDPR compliant IP handling come from. :)

Reply:
Thanks for clarifying. That says: "Logs should otherwise only include the first two octets of IPv4 addresses, or first three octets of IPv6 addresses". I was planning on keeping full IPs for a few days (else DoS attacks would be tricky to pinpoint), and then anonymising. But I guess we are probably OK pre-anonymising....it's not like I've ever needed them before!

However: @RabsRincon -- you're only anonymising the last octet of IPv4. The recommendation above is to anonymise the last TWO octets, it seems.